src/base/static/components/molecules/map-centerpoint.tsx

```diff
@@ -2,20 +2,19 @@
 import * as React from "react";
 import { connect } from "react-redux";
 import { css, jsx } from "@emotion/core";
+
 import { uiVisibilitySelector } from "../../state/ducks/ui";
-type StateProps = {
-  isMapCenterpointVisible: boolean;
-};
+import {
+  isMapDraggingOrZooming,
+  isMapDraggedOrZoomedByUser,
+} from "../../state/ducks/map";
 
-type Props = {
-  isMapDraggedOrZoomed: boolean;
+type StateProps = {
+  isMapDraggedOrZoomedByUser: boolean;
   isMapDraggingOrZooming: boolean;
-} & StateProps;
+};
 
-const MapCenterpoint: React.FunctionComponent<Props> = props => {
-  if (!props.isMapCenterpointVisible) {
-    return null;
-  }
```

Comment:
FYI this was added as a perf enhancement. It prevents MainMap from having to subscribe to the `isMapCenterpointVisible` state. Moving Redux subscriptions out of perf bottlenecks like MainMap and into *connected* components, instead of *controlled* components, is a pattern to reduce the rendering overhead of MainMap.

It's probably negligible either way, but just a heads up.

Reply:
I see-- thanks for the explanation. I'll revert this change in that case. I'd like to leverage every perf enhancement we can get to keep map rendering smooth.

Reply:
I hadn't realized this was something you changed in the viewport PR; I thought this was something I had overlooked previously...